modules/Microsoft.EventHub/namespaces/readme.md

```diff
@@ -515,8 +515,6 @@ module namespaces './Microsoft.EventHub/namespaces/deploy.bicep' = {
       "value": [
         {
           "name": "<<namePrefix>>-az-evh-x-001"
-        },
```

Comment:
Hey @BonnieMcb this is a good catch. But it actually needs to be fixed in the underlying test file: 
https://github.com/Azure/ResourceModules/blob/ac97d3b5d4485792b443ff3c2d12758a103abe0f/modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep#L89-L91
and then we have to re-run the `Set-ModuleReadMe` script to update the ReadMe that you suggested the original fix for.

If you need any support with that, let us know :)

Reply:
See other comment